### PR TITLE
update doc for podman

### PR DESCRIPTION
**Pull request by mengqiy**
fixes: https://github.com/GoogleContainerTools/kpt/issues/3258

### CODE REVIEW

File: site/book/01-getting-started/01-system-requirements.md [hunk 1]
```diff
@@ -13,9 +13,24 @@ $ kpt version
...
-kpt requires that you have [Docker] installed and configured.
+`kpt` requires you to have at least one of the following runtimes installed and configured.
+
+Environment variable can be used to control which container runtime to use. More
+details can be found in the reference documents for [`kpt fn render`](/reference/cli/fn/render/)
+and [`kpt fn eval`](/reference/cli/fn/eval/).
+
+### Docker
+
+Here is the [instructions][install-docker] to have Docker installed and configured.
```
Comment (droot) [inline site/book/01-getting-started/01-system-requirements.md:26]: Suggestion:

Please follow the [instructions] to install and configure Docker.

**Review by droot — APPROVED** [review]
Looks good. Have minor suggestions.

File: site/book/01-getting-started/01-system-requirements.md (hunk 1 above)
Comment (mengqiy) [reply site/book/01-getting-started/01-system-requirements.md]: Done

File: site/book/01-getting-started/01-system-requirements.md [hunk 2]
```diff
@@ -13,9 +13,24 @@ $ kpt version
...
+
+### Podman
+
+Here is the [instructions][install-podman] to have Podman installed and configured.
```
[the remark this answers is not in the document]
Comment (mengqiy) [reply site/book/01-getting-started/01-system-requirements.md]: Done

File: site/book/01-getting-started/01-system-requirements.md [hunk 3]
```diff
@@ -13,9 +13,24 @@ $ kpt version
 
 kpt requires that you have [Git] installed and configured.
 
-## Docker
+## Container Runtime
 
-kpt requires that you have [Docker] installed and configured.
+`kpt` requires you to have at least one of the following runtimes installed and configured.
+
+Environment variable can be used to control which container runtime to use. More
```
[the remark this answers is not in the document]
Comment (mengqiy) [reply site/book/01-getting-started/01-system-requirements.md]: Done

**Pull request merged**